Nimb3s.Automaton.DataAccess/JobData.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Nimb3s.Automaton.Pocos;
```

Comment:
nice, you got the gist of it on how to make db calls. lets move this logic into the [`Nimb3s.Automation.Core.Repositories.Sql](https://github.com/nimb3s/automaton/tree/main/src/Nimb3s.Automaton.Core.Repositories.Sql) library. 

you will want to add a new method to the [`JobStatusRepository`](https://github.com/nimb3s/automaton/blob/main/src/Nimb3s.Automaton.Core.Repositories.Sql/JobStatusRepository.cs). maybe called `GetDetailsAsync(Guid jobId)`